internal/client/datadog/client_test.go

```diff
@@ -69,27 +69,28 @@ var _ = Describe("DatadogClient", func() {
 			c.apiURL = "https://app.datadoghq.com/"
 			result, err := c.seriesURL()
 			Expect(err).To(BeNil())
```

Comment:
Can you assert the headers are correctly set?

Reply:
Done